src/pages/filter/ui/selected-group.tsx

```diff
@@ -17,6 +17,17 @@ export const SelectedGroup = observer((): ReactElement => {
 
   const { isRedactorMode } = filterStore
 
+  const getGroup = () => {
```

Comment:
Please don't use closure function here. You can use simple assignment with `group` variable.
Or if you want, you may create independent component e.g. <SelectedGroupPanel />